tools/runners/run-proxy.js

```diff
@@ -244,38 +244,169 @@ Object.assign(Proxy.prototype, {
 });
 
 function showErrorPage(res) {
-  // XXX serve an app that shows the logs nicely and that also
-  // knows how to reload when the server comes back up
+  // TODO: reload when the server comes back up
```

Comment:
Do you mind opening an issue and then track it here?

Reply:
Done #13849